modules/installation-vsphere-config-yaml.adoc

```diff
@@ -1,16 +1,18 @@
 // Module included in the following assemblies:
 //
-// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
-// * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
-// * installing/installing_vsphere/installing-vsphere.adoc
+// xref:../installing/installing_vsphere/installing-restricted-networks-vsphere.adoc[Installing a cluster on vSphere in a restricted network with user-provisioned infrastructure].
+
+// xref:../installing/installing_vsphere/installing-vsphere.adoc[Installing a cluster on vSphere with user-provisioned infrastructure]
```

Comment:
The previous format for these callouts was correct - these xrefs will still be checked by the build. Please remove them.

Reply:
Yeah, I'm just running an experiment to see if I can make such files easier to work with. I noticed an ifdef statement that a few writers were unable to fully explain if it was needed or not.